# Digital map of environment, the result of road, obstacles, object etc detection
nav_msgs/OccupancyGrid grid     # occupancy grid with obstacles